cluster-operator/src/main/java/io/strimzi/operator/cluster/model/EntityOperator.java

```diff
@@ -184,11 +186,14 @@ public Deployment generateDeployment() {
 
         Resources tlsSidecarResources = (tlsSidecar != null) ? tlsSidecar.getResources() : null;
 
+        TlsSidecarLogLevel tlsSidecarLogLevel = (tlsSidecar != null) ? tlsSidecar.getLogLevel() : TlsSidecarLogLevel.NOTICE;
```

Comment:
I've had weirdness in the past with field defaults when used with Jackson. If you don't specify a level explicitly in the YAML are you sure that `tlsSidecar.getLogLevel()` ends up as the default and not null?

Reply:
Yes it works I tried it. Btw it's a good point because in the `TlsSidecar` class I have:

```
private TlsSidecarLogLevel logLevel = TlsSidecarLogLevel.NOTICE;
```

but even `@DefaultValue("notice")` on the `getLogLevel` method.
If you had problems in the past maybe the `@DefaultValue` doesn't work but the field assignment is fixing it?

Reply:
which is the same for the `ActRule` class with `AclRuleType`